app/gui/controller/double-representation/src/project.rs

```diff
@@ -76,6 +76,11 @@ impl QualifiedName {
         }
     }
 
+    /// Return the qualified name of the "Standard.Base" library project.
+    pub fn of_standard_base_library() -> Self {
+        Self::from_segments("Standard", "Base").unwrap()
```

Comment:
Is this the only place in the whole codebase where any of the string literals "Standard" and "Base" appear? If so, they should be refactored to constants and placed in an easy to find place.